### PR TITLE
Use PHPStorm & PHP Inspections (EA Extended) on source code

### PR DESCRIPTION
**Pull request by machour**

### CODE REVIEW

File: controllers/ExtensionController.php [hunk 1]
```diff
@@ -359,7 +359,6 @@ public function actionFiles($id)
 
         return $this->render('files', [
             'model' => $model,
-            'files' => $model->downloads,
```
Comment (samdark) [inline controllers/ExtensionController.php]: Why?

File: models/ContentShareQuery.php [hunk 2]
```diff
@@ -17,23 +17,4 @@ public function forPublishing()
         return $this->andWhere(['status_id' => [ContentShare::STATUS_NEW, ContentShare::STATUS_FAILED]]);
     }
 
-    /**
-     * @inheritdoc
-     *
-     * @return ContentShare[]|array
-     */
-    public function all($db = null)
```
Comment (samdark) [inline models/ContentShareQuery.php]: Why?

File: models/ExtensionQuery.php [hunk 3]
```diff
@@ -47,21 +47,4 @@ public function excludeOfficial()
         return $this->andWhere("name NOT LIKE 'yiisoft/yii2-%'");
     }
 
-    /**
-     * @inheritdoc
-     * @return Extension[]|array
-     */
-    public function all($db = null)
```
Comment (samdark) [inline models/ExtensionQuery.php]: Why?

File: models/UserQuery.php [hunk 4]
```diff
@@ -13,22 +18,4 @@ public function active()
     {
         return $this->andWhere(['status' => User::STATUS_ACTIVE]);
     }
-
-    /**
-     * @inheritdoc
-     * @return User[]|array
-     */
-    public function all($db = null)
```
Comment (samdark) [inline models/UserQuery.php]: Why?

File: models/WikiQuery.php [hunk 5]
```diff
@@ -23,22 +28,4 @@ public function active()
     {
         return $this->andWhere(['status' => Wiki::STATUS_PUBLISHED]);
     }
-
-    /**
-     * @inheritdoc
-     * @return Wiki[]|array
-     */
-    public function all($db = null)
```
Comment (samdark) [inline models/WikiQuery.php]: Why?

File: models/ContentShareQuery.php (hunk 2 above)
Comment (machour) [reply models/ContentShareQuery.php]: it's just calling the parent method, why is it needed?

File: controllers/ExtensionController.php (hunk 1 above)
Comment (machour) [reply controllers/ExtensionController.php]: View 'files' do not use the "files" parameter

File: models/ContentShareQuery.php (hunk 2 above)
Comment (machour) [reply models/ContentShareQuery.php]: is it for IDE completion?

Comment (samdark) [reply models/ContentShareQuery.php]: It is.

Comment (machour) [reply models/ContentShareQuery.php]: Wouldn't a PHPDoc comment at the class level be better?
```
 * @method ContentShare[]|array all($db = null)
```

Comment (samdark) [reply models/ContentShareQuery.php]: Likely.

Comment (machour) [reply models/ContentShareQuery.php]: Done in https://github.com/yiisoft-contrib/yiiframework.com/pull/428/commits/9cb834e4823fd114b2083e05c197178b61d5a28c

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**